.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -45,3 +52,16 @@ body:
       description: Steps to reproduce
```

Comment:
```suggestion
      description: Include steps to reproduce, describe the behaviour you observe, and describe the behaviour you _expect_ to observe
```

Reply:
It might be helpful to break these into discrete fields in order to better guide users towards including all relevant info:
* Describe the bug [include any screenshots or relevant log output]
* Link to reproduction (optional)
* Steps to reproduce
* Expected result
* Actual result